STACpopulator/logging.py

```diff
@@ -1,6 +1,6 @@
 import datetime as dt
 import json
-import logging
+import logging.config
```

Comment:
There is (at least) `logging.getLevelName` reference in there, so `import logging` is needed as well.

Reply:
`import logging.config` adds the `logging` module to the local namespace as well (if it didn't we would see a failing test). Adding an additional `import logging` is redundant.